challenge_serving_drinks/src/challenge_serving_drinks/serving_drinks.py

```diff
@@ -0,0 +1,82 @@
+# ROS
+import smach
+
+# TU/e Robotics
+import robot_smach_states as states
+import robot_smach_states.util.designators as ds
+
+from robot_skills.classification_result import ClassificationResult
+from robot_skills.robot import Robot
+from robocup_knowledge import knowledge_loader
+
+# Challenge serving drinks
+from .drive_in import DriveIn
+from .serve_one_drink import ServeOneDrink
+
+# Knowledge
+CHALLENGE_KNOWLEDGE = knowledge_loader.load_knowledge("challenge_serving_drinks")
```

Comment:
Knowledge is imported at a lot of places. Would be much nicer if only done once.

Reply:
Will be reflected in the new PR (refactor)